editor/js/Viewport.js

```diff
@@ -152,8 +152,19 @@ function Viewport( editor ) {
 
 	function updateAspectRatio() {
 
-		camera.aspect = container.dom.offsetWidth / container.dom.offsetHeight;
-		camera.updateProjectionMatrix();
+		for ( const uuid in editor.cameras ) {
+
+			const camera = editor.cameras[ uuid ];
+
+			if ( camera.isPerspectiveCamera ) {
```

Comment:
Shouldn't all cameras be resized? The issue still happens with orthographic cameras.

Reply:
Correct, this PR only handle the perspective cameras' aspect. To keep consistent with this https://github.com/ycw/three.js/blob/7e97055eba802a9f4560943c0aafa9c0749bb7f0/editor/js/Viewport.js#L639-L643, should I also add `else if ( oroth camera ) { // TODO }` for clarification?

Reply:
This check should be removed and orthographic cameras resized like perspective cameras. The aspect needs to be honored in the `left` and `right` property otherwise the projection matrix is incorrect.

Reply:
I see, thanks for pointing out that, would you mind file a PR or add //TODO for now because ortho camera is not in my todo list :'(

Reply:
To make sure that we're on the same page, this is the issue we didn't fix:

https://github.com/mrdoob/three.js/assets/1063018/5d3686b9-ad39-48ae-86e3-b0a84bebaa0e